charts/capi-runtime-extensions/README.md

```diff
@@ -38,6 +38,8 @@ A Helm chart for capi-runtime-extensions
 | handlers.CalicoCNI.defaultTigeraOperatorConfigMap.name | string | `"tigera-operator"` |  |
 | handlers.CalicoCNI.enabled | bool | `true` |  |
 | handlers.ServiceLoadBalancerGC.enabled | bool | `true` |  |
+| handlers.http-proxy-inject.enabled | bool | `true` |  |
+| handlers.http-proxy-vars.enabled | bool | `true` |  |
```

Comment:
Maybe something like this for a consistent case
```suggestion
| handlers.HTTPProxyPatch.enabled | bool | `true` |  |
| handlers.HTTPProxyVars.enabled | bool | `true` |  |
```

Reply:
I've changed variable names as suggested.